web/src/Routes.tsx

```diff
@@ -17,33 +17,19 @@
 import React, {lazy, Suspense} from 'react';
 
 import Loading from 'components/common/Loading';
-import CustomerCommitCountProvider from 'components/customer/contexts/CommitCountContext';
+import CustomerRoutes from 'components/customer/Routes';
+import MerchantRoutes from 'components/merchant/Routes';
 import {Switch, Route} from 'react-router-dom';
 
-// Customer Pages
-const CustomerExplorePage = lazy(() => import('components/customer/explore'));
-// Merchant Pages
-const MerchantLandingPage = lazy(() => import('components/merchant/landing'));
-const MerchantSignUpPage = lazy(() => import('components/merchant/sign-up'));
-const MerchantSignInPage = lazy(() => import('components/merchant/sign-in'));
-const MerchantListingsPage = lazy(() => import('components/merchant/listings'));
-// Common Pages
-const ListingDetailsPage = lazy(() => import('components/listing-details'));
 // Design samples
 const DesignSamplesPage = lazy(() => import('components/design-samples'));
 
 const Routes: React.FC = () => (
   <Suspense fallback={<Loading />}>
     <Switch>
-      <Route exact path="/merchant" component={MerchantLandingPage} />
```

Comment:
I see you removed this landing page, is it expected?

Reply:
@ifhuang based on this comment (https://github.com/googleinterns/gpay-group-buy/pull/116#discussion_r453729049), I decided to just remove it for now. I will add that in later in another PR according to Caryn's suggestion.